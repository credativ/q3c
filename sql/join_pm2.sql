select count(*) from test_pm1 as a, test as b where q3c_join (
       a.ra+a.pmra/3600/1000*(2000-epoch),
       a.dec+a.pmdec/3600/1000*(2000-epoch),
       b.ra,b.dec,0.002);
select count(*) from test_pm1 as a, test as b where q3c_join(
       a.ra+a.pmra/3600/1000*(2000-epoch),
       a.dec+a.pmdec/3600/1000*(2000-epoch),
       b.ra,b.dec,0.004);
select count(*) from test_pm1 as a, test as b where q3c_join(
       a.ra+a.pmra/3600/1000*(2000-epoch),
       a.dec+a.pmdec/3600/1000*(2000-epoch),
       b.ra,b.dec,0.01);
select count(*) from test_pm1 as a, test as b where q3c_join(
       a.ra+a.pmra/3600/1000*(2000-epoch),
       a.dec+a.pmdec/3600/1000*(2000-epoch),
       b.ra,b.dec,0.02);
select count(*) from test_pm1 as a, test as b where q3c_join(
       a.ra+a.pmra/3600/1000*(2000-epoch),
       a.dec+a.pmdec/3600/1000*(2000-epoch),
       b.ra,b.dec,0.042);

select count(*) from test_pm1 as a, test as b where q3c_join_pm (a.ra,a.dec,a.pmra,a.pmdec,a.epoch,b.ra,b.dec,2000,100,0.002);
select count(*) from test_pm1 as a, test as b where q3c_join_pm(a.ra,a.dec,a.pmra,a.pmdec,a.epoch,b.ra,b.dec,2000,100,0.004);
select count(*) from test_pm1 as a, test as b where q3c_join_pm(a.ra,a.dec,a.pmra,a.pmdec,a.epoch,b.ra,b.dec,2000,100,0.01);
select count(*) from test_pm1 as a, test as b where q3c_join_pm(a.ra,a.dec,a.pmra,a.pmdec,a.epoch,b.ra,b.dec,2000,100,0.02);
select count(*) from test_pm1 as a, test as b where q3c_join_pm(a.ra,a.dec,a.pmra,a.pmdec,a.epoch,b.ra,b.dec,2000,100,0.042);
