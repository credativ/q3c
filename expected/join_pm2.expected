 count 
-------
   286
(1 row)

 count 
-------
  1209
(1 row)

 count 
-------
  6792
(1 row)

 count 
-------
 29671
(1 row)

 count  
--------
 133637
(1 row)

 count 
-------
   286
(1 row)

 count 
-------
  1209
(1 row)

 count 
-------
  6792
(1 row)

 count 
-------
 29671
(1 row)

 count  
--------
 133637
(1 row)

 count 
-------
   307
(1 row)

 count 
-------
  1210
(1 row)

 count 
-------
  6849
(1 row)

 count 
-------
 29704
(1 row)

 count  
--------
 133672
(1 row)

 count 
-------
   307
(1 row)

 count 
-------
  1210
(1 row)

 count 
-------
  6849
(1 row)

 count 
-------
 29704
(1 row)

 count  
--------
 133672
(1 row)

